### PR TITLE
feat(run/pubsub): add Cloud Run sample for Pub/Sub tutorial

### PR DESCRIPTION
**Pull request by drch-**
Example that logs a simple Pub/Sub message consistent with examples from other frameworks.
Provides the code to implement the tutorial at https://cloud.google.com/run/docs/tutorials/pubsub

Implemented with .NET 6 and Minimal APIs.

### CODE REVIEW

File: run/pubsub/Run.Samples.Pubsub.MinimalApi/Run.Samples.Pubsub.MinimalApi.csproj [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```
Comment (amanda-tarafa) [inline run/pubsub/Run.Samples.Pubsub.MinimalApi/Run.Samples.Pubsub.MinimalApi.csproj:4]: The repo still does not support .NET 6, but I'll work on that this week, see #1257

File: run/pubsub/Run.Samples.Pubsub.MinimalApi/Program.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        app.Logger.LogWarning("Bad Request: Invalid Pub/Sub message format.");
+        return Results.BadRequest();
+    }
+
+    var data = Convert.FromBase64String(envelope.Message.Data);
+    var target = System.Text.Encoding.UTF8.GetString(data);
+
+    app.Logger.LogInformation($"Hello {target}!");
+
+    return Results.NoContent();
```
Comment (amanda-tarafa) [inline run/pubsub/Run.Samples.Pubsub.MinimalApi/Program.cs:37]: I'm not sure about this, but will this acknowledge the message, does it need to?

**Review by amanda-tarafa — CHANGES_REQUESTED** [review]
Overall looks good.

For tests to be run by CI (after I've fixed #1257 ), you need to add this script https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/main/kms/api/runTests.ps1 on your top level folder and add a dependency on JUnitTestLogger on your test project.

**Comment by amanda-tarafa** [comment]
Oh, I forgot to say, you are missing the region tags, which will allow you to include the sample code on the documentation you linked above. See the Java sample [here](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/f19993c6ce5dfccd484bd3eb78e43f5fa6f29844/run/pubsub/src/main/java/com/example/cloudrun/PubSubApplication.java#L19), for instance, on which region tags are being used. You will need to change the documentation as well so that the C# sample is included. Ping me internally if I can help with that.

File: run/pubsub/Run.Samples.Pubsub.MinimalApi/Program.cs (hunk 2 above)
Comment (drch-) [reply run/pubsub/Run.Samples.Pubsub.MinimalApi/Program.cs:37]: Yes - the use case in the tutorial uses push notifications and this status code will acknowledge the message.

**Comment by drch-** [comment]
> You will need to change the documentation as well so that the C# sample is included. Ping me internally if I can help with that.

Thanks!  I've added the tags and will ping you for the other steps.

File: run/pubsub/runTests.ps1 [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Google LLC
```
Comment (amanda-tarafa) [inline run/pubsub/runTests.ps1:1]:
```suggestion
# Copyright 2022 Google LLC
```

**Review by amanda-tarafa — APPROVED** [review]
LGTM.

I'll let you know as soon as I've address #1257

**Comment by amanda-tarafa** [comment]
@drch- I've addressed #1257 and CI now supports .NET 6, we should be ready to merge this if tests are green and you have included the sample in docs. Let me know.

**Pull request merged**